server/src/main/java/com/objectcomputing/checkins/services/permissions/Permission.java

```diff
@@ -8,12 +8,14 @@
 @JsonSerialize(using = PermissionSerializer.class)
 public enum Permission {
   CAN_VIEW_FEEDBACK_REQUEST("View feedback requests", "Feedback"),
+  CAN_DENY_FEEDBACK_REQUEST("Deny feedback requests", "Feedback"),
   CAN_CREATE_FEEDBACK_REQUEST("Create feedback requests", "Feedback"),
   CAN_DELETE_FEEDBACK_REQUEST("Delete feedback requests", "Feedback"),
   CAN_CREATE_KUDOS("Create kudos", "Feedback"),
   CAN_ADMINISTER_KUDOS("Administer kudos", "Feedback"),
   CAN_VIEW_FEEDBACK_ANSWER("View feedback answers", "Feedback"),
   CAN_SEND_EMAIL("Send email", "Feedback"),
+  CAN_SEND_NOTIFICATION("Send notification", "Feedback"),
```

Comment:
I would make `notifications` plural and create a new category for EMAIL and NOTIFICATIONS above called "Notifications" and use that instead of "Feedback" here.

Reply:
Complete.